app/dashboard/templates/shared/nav_auth.html

```diff
@@ -41,7 +41,7 @@
           {% trans "Fund Issue" %}
         </a>
         <a class="dropdown-item" href="{% url "grants:profile" %}">
-          <i class="fas fa-money-check"></i>
+          <i class="fas"><img src="{% static "v2/images/grants/IconMyGrants.svg" %}"></i>
```

Comment:
do we need the `fas` class? @usmanmuhd

Reply:
same comment for kudos one. I merged that [PR](https://github.com/gitcoinco/web/pull/2971/files#r237012001).
Could you open a new one thought?

Reply:
It is required to keep the icon aligned in the same way as the other icons.